src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -158,6 +158,7 @@
             durationCell.className = 'sf-ajax-request-duration';
             durationCell.textContent = 'n/a';
             row.appendChild(durationCell);
+            request.updateDuration = setInterval(request.updateDuration, 100, durationCell);
```

Comment:
using `request.updateDuration`  to store both the callback and the timer identifier after that looks wrong to me. It will make it much easier to introduce bugs by mistake in the future.